app/uk/gov/hmrc/childcarecalculatorfrontend/views/freeHoursInfo.scala.html

```diff
@@ -54,11 +54,12 @@ <h2 class="heading-medium">@messages(s"freeHoursInfo.heading2")</h2>
 
 
     @if(hasChildcareCosts == false) {
-        <p>@messages("freeHoursInfo.no.childcare.para")</p>
+        <p>@messages("freeHoursInfo.no.childcare.para") <a href="">@messages("freeHoursInfo.no.childcare.link")</a>. @messages("freeHoursInfo.no.childcare.para.end")</p>
```

Comment:
Please update in the reply of this comment when you are done with all of your changes

Reply:
i dont have the link to put in here... but it should be linked up

Reply:
The references to routes was wrong, hence why we had no access to routes. Fixing this now. Thanks.